GOVERNANCE.md

```diff
@@ -3,43 +3,58 @@ the Benevolent dictator governance model by Ross
 Gardler and Gabriel Hanganu licensed under a Creative Commons 
 Attribution-ShareAlike 4.0 International License. -->
 
-# Overview
-This project is led by a benevolent dictator and managed by a core team of developers and a large community of contributors and users. That is, the community and core developers actively contribute to the day-to-day maintenance of the project, but the general strategic line is drawn by the benevolent dictator. In case of disagreement, they have the last word.
+# Roles and responsibilities
 
-# Roles And Responsibilities
-## Benevolent dictator (Hadley Wickham, @hadley)
-The job of the benevolent dictator is to set the strategic objectives of the project and communicate these clearly to the community, ensuring that the project survives in the long term. 
+This project has a large community __users__ and __contributors__, a team of __core developers__, and a __project lead__.
 
-## [Core Developers](https://github.com/orgs/tidyverse/teams/ggplot2)
-Core developers are contributors who have made several valuable contributions to the project and are now relied upon to write code directly to the repository and screen the contributions of others. Core developers are empowered to merge pull requests after careful review. Core developers have no authority over the overall direction of the project, however it is their job to help develop or elicit appropriate contributions to the project. As a matter of policy, core developers, even if no longer active, are listed (chronologically) in [package authors](https://ggplot2.tidyverse.org/authors.html).
+## Users
+
+Anyone who creates visualisations with ggplot2 is a user. They are the most important members of the community: without them, the project would have no purpose. 
 
-An invitation to join the core team will be extended to contributors who have made a major contributions as a contributor. For example, they have added a major new feature or have a consistent pattern of smaller contributions over an extended period of time. Any existing core developer can propose a contributor be invited to the core team, pending confirmation by existing authors.
+Users are encouraged to participate in the life of the project and the community as much as possible. User contributions enable the project team to ensure that they are satisfying the needs of those users. Common user activities include (but are not limited to):
```

Comment:
User contributions help ensure that the project team is satisfying users' needs.